### DISCUSSION TITLE
Issue with Search Filters

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3499] [Q&A] [answered]
Search results are throwing error when search filter exceed 65,000 count (see below). Is it the max number that is allowed?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/159083804/df3ae14d-a867-42e5-b136-152a2aa16168)

**Comment by wobba** [comment, marked as answer]
API related, not web part. So this is not the place to ask as this project is bound by any API limitation.

Also returning 65.000 unique values is likely slow and not very user friendly.

**Reply by ellanrk** [reply]
Thanks for the quick response!
